run_dir/static/js/worksets.js

```diff
@@ -215,6 +242,41 @@ function init_listjs2() {
     } );
 }
 
+// Initialize sorting and searching javascript plugin
+function init_list_closed_ws() {
+    // Setup - add a text input to each footer cell
+    $('#closed_ws_table tfoot th').each( function () {
+      var title = $('#closed_ws_table thead th').eq( $(this).index() ).text();
+      $(this).html( '<input size=10 type="text" placeholder="Search..." />' );
+    } );
+
+    var table = $('#closed_ws_table').DataTable({
+      "paging":false,
+      "info":false,
+      "order": [[ 0, "desc" ]]
+    });
+
+    //Add the bootstrap classes to the search thingy
+    if($('#workset_table_filter').length){
+      $('#workset_table_filter').hide();
+    }
+    $('div.dataTables_filter input').addClass('form-control search search-query');
+    $('#closed_ws_table_filter').addClass('form-inline pull-right');
+    $("#closed_ws_table_filter").appendTo("h1");
+    $('#closed_ws_table_filter label input').appendTo($('#closed_ws_table_filter'));
+    $('#closed_ws_table_filter label').remove();
+    $("#closed_ws_table_filter input").attr("placeholder", "Search..");
```

Comment:
Should be three '.' right?

Reply:
Just copied it from another part of the code, yes it looks better with three dots ^^